egs/wsj/s5/utils/mkgraph.sh

```diff
@@ -20,10 +20,9 @@ loopscale=0.1
 
 remove_oov=false
 
-for x in `seq 6`; do
-  [ "$1" == "--mono" ] && context=mono && shift;
-  [ "$1" == "--left-biphone" ] && context=lbiphone && shift;
-  [ "$1" == "--quinphone" ] && context=quinphone && shift;
+for x in `seq 4`; do
+  [ "$1" == "--mono" -o "$1" == "left-biphone" -o "$1" == "--quinphone" ] && shift && \
+    echo "WARNING: The '--mono, --left-biphone and --qunphone' options are deprecated and ignored. Please delete it!"
```

Comment:
please spell quinphone right.  Also remove the words "Please delete it!".  Firstly, I don't think this adds useful information; secondly, the exclamation mark '!' is problematic in interactive shells,  if not enclosed in single quotes; it can cause crashes.